app/qml/editor/inputtextedit.qml

```diff
@@ -107,6 +116,15 @@ Item {
       }
     }
 
+    //! Commit value if has changed when widget gets out of the FeatureForm (ListView) viewport
+    Component.onDestruction: {
+      if ( textArea.activeFocus ) {
+        if ( value !== textArea.text ) {
+          valueChanged( textArea.text, textArea.text === undefined )
```

Comment:
I know we are using the comparison to undefined all over the place with `valueChanged` signal, but I am thinking whether the empty string would not make more sense

```suggestion
          valueChanged( textArea.text, textArea.text === "" )
```
Or maybe better:
```suggestion
          valueChanged( textArea.text, textArea.text === undefined || textArea.text === "" )
```

Reply:
The second parameter of `ValueChange` signal is `bool isNull`. Handling of null values is not handled properly as I can recall (issues can be easily found by searching "NULL"). I would like to resolve those issues, but not in this PR.